generator/views/lookml_utils.py

```diff
@@ -209,3 +214,29 @@ def get_distinct_vals(bq_client: bigquery.Client, table: str, column: str):
 def slug_to_title(slug):
     """Convert a slug to title case."""
     return slug.replace("_", " ").title()
+
+
+# Map from view to qualified references {view: [[project, dataset, table],]}
+BQViewReferenceMap = Dict[str, List[List[str]]]
```

Comment:
it looks like all the uses of BQViewReferenceMap are in `Dict[str, BQViewReferenceMap]`, why not have the type include that outer dict? i.e.:
```suggestion
BQViewReferenceMap = Dict[str, Dict[str, List[List[str]]]]
```

Reply:
Yeah this makes sense, I'll update along with the description comment